src/main/java/com/gameshop/Controller/AnalysController.java

```diff
@@ -0,0 +1,63 @@
+package com.gameshop.Controller;
+
+import com.gameshop.dto.GoodsWithPromoCode;
+import com.gameshop.dto.OrderDto;
+import com.gameshop.entity.*;
+import com.gameshop.service.AnalystService;
+import com.sun.javafx.collections.MappingChange;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.format.annotation.DateTimeFormat;
+import org.springframework.web.bind.annotation.*;
+
+import javax.validation.Valid;
+import java.time.LocalDate;
+import java.util.Date;
+import java.util.List;
+import java.util.Map;
+
+@RestController
+@RequestMapping("/analys")
+public class AnalysController {
+
+    @Autowired
+    AnalystService analystService;
+
+    @GetMapping("/getReportOfDay")
+    public List<ReportOfDay> getReportOfDay(@RequestParam(value = "start") @DateTimeFormat(iso = DateTimeFormat.ISO.DATE) LocalDate start, @RequestParam(value = "end") @DateTimeFormat(iso = DateTimeFormat.ISO.DATE) LocalDate end) {
+        return analystService.getReportOfDay(start, end);
+    }
+
+
+    @GetMapping("/getDateReport")
+    public DateReport searchOrder(@RequestParam(value = "start") @DateTimeFormat(iso = DateTimeFormat.ISO.DATE) LocalDate start, @RequestParam(value = "end") @DateTimeFormat(iso = DateTimeFormat.ISO.DATE) LocalDate end) {
```

Comment:
Don't use words "get, retrieve, add, delete" and others in REST controllers
You have @GetMapping annotation.